pyerrors/misc.py

```diff
@@ -183,3 +183,18 @@ def _assert_equal_properties(ol, otype=Obs):
             if hasattr(ol[0], attr):
                 if not getattr(ol[0], attr) == getattr(o, attr):
                     raise Exception(f"All Obs in list have to have the same state '{attr}'.")
+
+
+def obsval(o):
```

Comment:
I think the build in `float` function should be able to do this as well.

Reply:
```python
float(o) is o.value
```

Reply:
Very good point. In general terms, `float(o)` could not cope with `o` being `None`, so `obsval` would be a bit more flexible. But that should not matter in this case. I can remove the function.